blazor/datagrid/scrolling.md

```diff
@@ -458,4 +458,64 @@ The [GridColumns](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Grids.
 
 > If frozen columns are not specified, the frozen column separator will be displayed at the left and right ends. You can dynamically change the frozen columns by dragging the column separator.
 
-![Add or Remove Frozen Blazor DataGrid Columns by Dragging the Column Separator](./images/blazor-datagrid-drag-and-drop-the-column-separator.gif)
\ No newline at end of file
+![Add or Remove Frozen Blazor DataGrid Columns by Dragging the Column Separator](./images/blazor-datagrid-drag-and-drop-the-column-separator.gif)
+
+## Customize grid scroll bar
+
+The Grid component uses the native browser scroll bar to scroll through the content when the content is larger than the Grid. If you want to customize the appearance of the scroll bar, you need to refer to [this](https://css-tricks.com/almanac/properties/s/scrollbar/).
```

Comment:
The Grid component uses the native browser scroll bar to scroll through the content when the content is larger than the Grid. Refer to [this](https://css-tricks.com/almanac/properties/s/scrollbar/) to customize the appearance of the scroll bar.